README.md

```diff
@@ -93,9 +93,8 @@ In the future we should register this method in the `apps.py` file of the app, b
 #### Versions
 
 We currently have 2 versions of the API.
```

Comment:
We hebben m niet meer. Dus ik zou iets zeggen zoals `We used to have 2 versions...; The first version was the `v1` API, which has been removed.